test/unit/puppetlabs/services/master/master_core_test.clj

```diff
@@ -91,89 +94,94 @@
                               "/v3/environment_classes?environment=~"))))))
       (testing "calculates etag properly for response payload"
         (is (= (etag {"/one/file"
-                      [
-                       {"name" "oneclass",
-                        "params" [
-                                  {"name" "oneparam",
-                                   "type" "String",
-                                   "default_literal" "'literal'",
-                                   "default_source" "literal"},
-                                  {"name" "twoparam",
-                                   "type" "Integer",
-                                   "default_literal" "3",
-                                   "default_source" "3"}]
-                        },
-                       {"name" "twoclass"
-                        "params" []}],
-                      "/two/file" []})
+                      (map-with-classes
```

Comment:
The comparison structures in this test are basically the same as before - just wrapped in a hash map with a `:classes` key now, since that's the format the request down to the JRuby layer will now always return.